### ISSUE TITLE
Build is stuck

### ISSUE THREAD

**Issue by frankbenoit** [diffplug/goomph-eclipse-plugin#2]
Hi

i try to run this example. I use Gradle 4.0.1 and java 8 u131

> Configure project :target.p2
p2AsMaven eclipse-deps is dirty.
Initalizing maven group eclipse-deps from p2
Only needs to be done once, future builds will be much faster
p2AsMaven eclipse-deps installing from p2
Buildfile: <...>\Temp\goomph-ant-build8459310540695597019.xml
[p2.mirror] SLF4J: Class path contains multiple SLF4J bindings.
[p2.mirror] SLF4J: Found binding in [jar:file:/<...>/.gradle/caches/4.0.1/generated-gradle-jars/gradle-api-4
.0.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[p2.mirror] SLF4J: Found binding in [jar:file:/C:/Gradle/lib/gradle-logging-4.0.1.jar!/org/slf4j/impl/StaticLoggerBinder
.class]
[p2.mirror] SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[p2.mirror] SLF4J: Actual binding is of type [org.gradle.internal.logging.slf4j.OutputEventListenerBackedLoggerContext]
<=------------> 14% CONFIGURING [2m 26s]
> root project

This seems to hang, it tried to run it for several minutes. No network activity either.

**Comment by nedtwigg** [comment]
First run is pretty slow, needs to do a lot of bootstrapping.  12 minutes on my machine.  Steps I took:

- Fresh clone on a macbook
- `gradlew jar`
- Succeeded 12 minutes later

Later runs are much faster.

**Comment by frankbenoit** [comment]
ok, having now for 30min running, still no feedback, I leave it running ... 
It would be helpful, to have some indicator about proper activity.

**Comment by frankbenoit** [comment]
After 43min it finished with error:
c:\Temp\goomph-eclipse-plugin>gradle cleanIde ide
Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :target.p2
p2AsMaven eclipse-deps is dirty.
Initalizing maven group eclipse-deps from p2
Only needs to be done once, future builds will be much faster
p2AsMaven eclipse-deps installing from p2
Buildfile: ...\Temp\goomph-ant-build4462108673686634566.xml
[p2.mirror] SLF4J: Class path contains multiple SLF4J bindings.
[p2.mirror] SLF4J: Found binding in [jar:file:/.../.gradle/caches/4.0.1/generated-gradle-jars/gradle-api-4
.0.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[p2.mirror] SLF4J: Found binding in [jar:file:/C:/Gradle/lib/gradle-logging-4.0.1.jar!/org/slf4j/impl/StaticLoggerBinder
.class]
[p2.mirror] SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[p2.mirror] SLF4J: Actual binding is of type [org.gradle.internal.logging.slf4j.OutputEventListenerBackedLoggerContext]
[p2.mirror] Messages while mirroring artifact descriptors.
BUILD SUCCESSFUL

BUILD SUCCESSFUL
Total time: 41 minutes 53 seconds
p2AsMaven eclipse-deps creating runnable repo
p2AsMaven eclipse-deps creating maven repo
p2AsMaven eclipse-deps is complete.

Download https://plugins.gradle.org/m2/com/diffplug/gradle/spotless/spotless/1.3.3/spotless-1.3.3.pom
Download https://plugins.gradle.org/m2/com/diffplug/freshmark/freshmark/1.3.0/freshmark-1.3.0.pom
Download https://plugins.gradle.org/m2/com/diffplug/durian/durian/3.2.0/durian-3.2.0.pom
Download https://plugins.gradle.org/m2/args4j/args4j/2.32/args4j-2.32.pom
Download https://plugins.gradle.org/m2/com/diffplug/jscriptbox/jscriptbox/3.0.0/jscriptbox-3.0.0.pom
Download https://plugins.gradle.org/m2/args4j/args4j-site/2.32/args4j-site-2.32.pom
Download https://plugins.gradle.org/m2/org/kohsuke/pom/14/pom-14.pom
Download https://plugins.gradle.org/m2/com/diffplug/freshmark/freshmark/1.3.0/freshmark-1.3.0.jar
Download https://plugins.gradle.org/m2/com/diffplug/durian/durian/3.2.0/durian-3.2.0.jar
Download https://plugins.gradle.org/m2/com/diffplug/jscriptbox/jscriptbox/3.0.0/jscriptbox-3.0.0.jar
Download https://plugins.gradle.org/m2/args4j/args4j/2.32/args4j-2.32.jar
Download https://plugins.gradle.org/m2/com/diffplug/gradle/spotless/spotless/1.3.3/spotless-1.3.3.jar

FAILURE: Build failed with an exception.

* What went wrong:
Task 'cleanIde' not found in root project 'goomph-eclipse-plugin'. Some candidates are: 'clean'.

* Try:
Run gradle tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info
or --debug option to get more log output.

BUILD FAILED in 43m 0s

**Comment by nedtwigg** [comment]
> It would be helpful, to have some indicator about proper activity.

Agreed!  Unfortunately the long waits are inside of eclipse utilities, so it's difficult for us to fix this inside of Goomph.

> finished with error

Don't run `cleanIde` and `ide` in the same command.  Does that fix it?

**Comment by frankbenoit** [comment]
The 'ide' does work.
The 'cleanIde' stops with error.
